sdk/core/src/main/java/com/klaviyo/core/KlaviyoException.kt

```diff
@@ -12,3 +12,20 @@ abstract class KlaviyoException(final override val message: String) : Exception(
 
     private fun log() = Registry.log.error(message, this)
 }
+
+/**
+ * Safely invoke a function and log KlaviyoExceptions rather than crash
+ */
+inline fun <T> safeCall(block: () -> T): T? {
```

Comment:
@ndurell and @kennyklaviyo I think this is a more appropriate place for these wrapper functions, lmk what you think